### PR TITLE
Option page

### PR DESCRIPTION
**Pull request by yogesh-**
Added the following features:
- Options to navigate to respective quiz pages
- favicon
- Link to logo
- created a database for questions and answers

image

### CODE REVIEW

File: src/pages/options/option.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    font-size: 2rem;
+  }
+
+  .lang-options {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    flex-wrap: wrap;
+    border: 2px solid pink;
```
Comment (zoyatanweer) [inline src/pages/options/option.css]: use var for color

File: src/pages/quiz/css-quiz.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Navbar } from "../../components/nav/nav";
+// import { CssQuestions } from "../../QuizDB/quizDb";
```
Comment (zoyatanweer) [inline src/pages/quiz/css-quiz.jsx:3]: remove commented code if not necessary

File: src/pages/quiz/js-quiz.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Navbar } from "../../components/nav/nav";
+// import { JavaScriptQuestions } from "../../QuizDB/quizDb";
```
Comment (zoyatanweer) [inline src/pages/quiz/js-quiz.jsx:3]: remove commented code if not required

**Review by zoyatanweer — APPROVED** [review]
- code looks great. 
- good to merge with suggested changes.
- great UI

**Review by Kothalkarpriya — APPROVED** [review]
Everything looks clean and good. You can merge the changes

**Pull request merged**